source/_integrations/system_bridge.markdown

```diff
@@ -29,18 +29,29 @@ You will need your API key. This can be found and configured in the application'
 
 This integration provides the following sensors:
 
-| Name             | Description                                         |
-| ---------------- | --------------------------------------------------- |
-| BIOS Version     | Version of your system's BIOS                       |
-| Battery          | Battery level of the device                         |
-| CPU Speed        | The current CPU speed                               |
-| CPU Temperature  | The current temperature of the device               |
-| Filesystem(s)    | Space used for each drive letter / filesystem mount |
-| Memory Free      | Memory (RAM) free in GB                             |
-| Memory Used      | Memory (RAM) used in GB                             |
-| Memory Used %    | Memory (RAM) % used                                 |
-| Operating System | Version information of the Operating System         |
-| Load             | System load percentage                              |
+| Name                   | Description                                         |
```

Comment:
Since this list exists I think it makes sense to split it based on which sensors are enabled by default.
The integration also provides binary sensors, but those are not mentioned.